tests/Makefile

```
@@ -9,4 +9,4 @@ PYTEST_LOGGING ?= DEBUG
 
 .PHONY: test
 test:  ## Run Pytest in verbose mode with CLI report only for all tests. By default only unit and system, but can be changed by declaring  TEST_PATH variable and its value
-	PYTEST_LOG_LEVEL=$(PYTEST_LOGGING) pytest $(TEST_OPT) $(REPORT) $(COVERAGE) --log-cli-level=$(CLI_LOGGING) -m '$(TAG)' $(TESTS)
+	export PYTEST_LOG_LEVEL=$(PYTEST_LOGGING) && pytest $(TEST_OPT) $(REPORT) $(COVERAGE) --log-cli-level=$(CLI_LOGGING) -m '$(TAG)' $(TESTS)
```

Not sure if you are trying to make this work but I have not find anything on the `PYTEST_LOG_LEVEL` environment variable. Neither in the doc or in the code of pytest. I would suggest the following changes.
```suggestion
	pytest $(TEST_OPT) $(REPORT) $(COVERAGE) --log-cli-level=$(CLI_LOGGING) --log-level=$(PYTEST_LOGGING) -m '$(TAG)' $(TESTS)
```

---

Ok I've just seen where it is used: for a custom logger.
I do not know what's the best way to handle logging in pytest, need to experiment first.

---

It is defined under `tests/lib/helpers.py`

```python
log_level = logging.getLevelName(os.environ.get('PYTEST_LOG_LEVEL', 'DEBUG'))
```